addons/mail/data/mail_templates_email_layouts.xml

```diff
@@ -177,12 +177,12 @@
                     <t t-esc="access_name"/>
                 </a>
             </div>
-            <t t-if="'user_id' in record and record.user_id and not record.env.user._is_superuser() and signature != ''">
+            <t t-if="email_add_signature and 'user_id' in record and record.user_id and not record.env.user._is_superuser()">
```

Comment:
Strange that "email_add_signature" and the emptiness of the signature are not checked at the same level.
Seems previous code was attempting to check that `signature != ''`.

Do we want to change that to include the "Best regards," part even if there is no signature on the user?

Reply:
Not untrue, like nobody says.